src/components/PeerReviewPage.js

```diff
@@ -30,29 +30,27 @@ class PeerReview extends React.Component {
     try {
       const group = await groupManagementService.getByStudent()
       if (group) {
-        if (group.id !== -100) {
```

Comment:
Why is the -100 not needed anymore? Does the backend still serve -100? Is it needed anywhere? If not, can it be removed from the backend?

Reply:
Backend no longer serve -100. Therefore this check can safely be removed. Backend will now serve null group which is checked by the line above.